strings/src/main/res/values/strings.xml

```diff
@@ -1111,6 +1111,8 @@
 
     <!-- Error shown when a user tries to open a form that's in a different project -->
     <string name="wrong_project_selected_for_form">To open this form, you must first open ODK Collect and switch to the project that contains it.</string>
+    <!-- Error shown when a user tries to open a form when there are no projects -->
+    <string name="app_not_configured">ODK Collect has not been configured. Try to open ODK Collect and configure it.\n\nIf you tapped on a shortcut, you might need to recreate it after configuring ODK Collect.</string>
```

Comment:
@lognaturel would appreciate a sanity check on this copywriting.

Reply:
I think it’s ok! This only happens when there are 0 projects, right? Not also when a deleted project is referenced? If it also includes the case where a deleted project is referenced thrn I’d want to edit it a bit.

Reply:
Yeah, you're right: at the moment this only shows for the 0 projects case.